source/includes/getting-started/_lets-get-started.html.md

```diff
@@ -35,84 +75,104 @@ bonsai configure # Follow the instructions to retrieve your key, and enter it wh
+### Create from Command Line
 
 ```shell
 bonsai create myMountainCarBrain
 ```
+You may give you brain a name with `bonsai create BRAIN_NAME` but there is no option for a description with this method.
 
-## Load sample Inkling into your BRAIN
+# Train a BRAIN
 
-Load our sample Inkling file for the mountain car simulation.
+## Load sample Inkling into your BRAIN
 
 ```shell
 bonsai load
 ```
 
-## Train
+Load our sample Inkling file for the Mountain Car simulation.
 
-Start the training mode for your BRAIN.
+## Start Training Mode
 
 ```shell
 bonsai train start
 ```
 
-## Connect the simulator for training
+Almost there! Time to tell the AI engine to prepare a new BRAIN version for training. Start the training mode for your BRAIN.
 
-Connect the simulator to the BRAIN. Training begins automatically. If the simulator gets disconnected, training resumes from the same point when the simulator is reconnected.
+## Connect the simulator for training
 
 ```shell
 python mountaincar_simulator.py --train-brain=myMountainCarBrain --headless
 ```
 
-## View your BRAIN training status
+Everything is in place, it's time to start the simulator and test things out. To do this, call Python and then the simulator file. Training will begin automatically and if the simulator gets disconnected, training resumes from the same point when the simulator is reconnected if it's within an hour of the disconnect. Remember, as mentioned in [2.3 What is a simulator?][7] if you want your training to take place overnight, make sure that your computer won't go to sleep and disconnect for more than an hour or that training time will be lost!
 
-View your BRAIN's training status as it trains on the simulator.
+**Note:** we use headless here to indicate we don't need to see a graphical display from the simulator; if you'd like to see it and watch the simulator learn, omit this option. You'll need to remember to close this window when you're done with training before you can stop the training mode.
 
- **Go to [beta.bons.ai][5].**
+## View your BRAIN training status
 
-## Close the simulator
+View your BRAIN's training status as it trains on the simulator by going to the BRAIN's Dashboard page on [beta.bons.ai][5]. Training Mountain Car takes about an hour to get sufficient training to beat the game most of the time. If you want flawless victories each time the simulator can take up to 2 hours before you'll see the graph level out.
 
-Training mountain car takes about 45 minutes.
+![Fully Trained BRAIN][16]
 
-**Stop the simulator by closing it.**
+There is no automatic ending to training, you can train forever, but there will be diminishing returns after a couple of hours. You can play around with training for 15 mins, 30 mins, 1 hour, etc and use your BRAIN to see how well it plays each time! It takes about 700 episodes to train the BRAIN correctly. Our ideal target is an average reward of less than -195 or higher over 100 consecutive episodes.
```

Comment:
I made a comment in the docs to update this section when we have multiple concepts.